plugins/abnormal_security/icon_abnormal_security/actions/get_cases/action.py

```diff
@@ -13,6 +13,8 @@ def __init__(self):
     def run(self, params={}):
         return {
             Output.CASES: self.connection.api.get_cases(
-                from_date=params.get(Input.FROM_DATE, None), to_date=params.get(Input.TO_DATE, None)
+                from_date=params.get(Input.FROM_DATE, None),
+                to_date=params.get(Input.TO_DATE, None),
+                filter_key=params.get(Input.FILTER_KEY, None),
```

Comment:
should this have a default if we don't require it as an input? Before it seemed to default to `receivedTime`, or maybe we pull the default from the schema in which case we don't need `None`. Don't need 'None' for the other input params either.

Reply:
this has been updated to use `lastModifiedTime` as the default here, `receivedTime` no longer works for this endpoint so we are now giving the user options to select the field themselves